safenode/src/client/api.rs

```diff
@@ -165,11 +165,12 @@ impl Client {
     }
 
     pub(crate) async fn send_to_closest(&self, request: Request) -> Result<Vec<Result<Response>>> {
-        info!("Sending {:?} to the closest peers.", request.dst());
+        println!("Getting the closest peers to {:?}.", request.dst());
```

Comment:
Is this change from logging to println intentional to keep ? (same as below).

Reply:
Yes. The logging was previously removed from client so none of this was visible.